Assets/MRTK/SDK/Experimental/Inspectors/ScrollingObjectCollection/ScrollingObjectCollectionInspector.cs

```diff
@@ -276,7 +278,7 @@ public override void OnInspectorGUI()
 
                     if (!CheckForStandardShader(nodeTransform.GetComponentsInChildren<Renderer>()))
                     {
-                        Debug.LogWarning(nodeTransform.name + " has a renderer that is not using " +  StandardShaderUtility.MrtkStandardShaderName + ". This will result in unexpected results with ScrollingObjectCollection");
+                        Debug.LogWarning(nodeTransform.name + " has a renderer that is not using " + StandardShaderUtility.MrtkStandardShaderName + ". This will result in unexpected results with ScrollingObjectCollection");
                     }
```

Comment:
This warning is printed with every foldout in the Scrolling Object Collection example scene because the PressableButtonHL2 does not use the mrtk standard shader, we might need to consider modifying the example scene if there is a possibility of unexpected behavior.

```
PressableButtonHoloLens2_32x96 (4) has a renderer that is not using Mixed Reality Toolkit/Standard. This will result in unexpected results with ScrollingObjectCollection
UnityEngine.Debug:LogWarning(Object)
Microsoft.MixedReality.Toolkit.Experimental.Inspectors.ScrollingObjectCollectionInspector:OnInspectorGUI() (at Assets/MRTK/SDK/Experimental/Inspectors/ScrollingObjectCollection/ScrollingObjectCollectionInspector.cs:281)
UnityEngine.GUIUtility:ProcessEvent(Int32, IntPtr)
```
Repro:
![ScrollingObjExampleWarnings](https://user-images.githubusercontent.com/53493796/88831529-529b0200-d184-11ea-9e28-cabe7c5b5bf9.gif)